lib/praxis/action_definition.rb

```diff
@@ -188,32 +188,69 @@ def description(text = nil)
       @description
     end
 
+    def self.url_description(route:, params_example:, params: )# TODO: what context to use?, context: [r.id])
```

Comment:
Looks like there's no reason to pass a context in, in the first place. The method takes an example object already, nothing is generated inside.